core/math/basis.cpp

```diff
@@ -763,7 +763,7 @@ Basis::operator String() const {
 
 Quaternion Basis::get_quaternion() const {
 #ifdef MATH_CHECKS
-	ERR_FAIL_COND_V_MSG(!is_rotation(), Quaternion(), "Basis must be normalized in order to be casted to a Quaternion. Use get_rotation_quaternion() or call orthonormalized() instead.");
+	ERR_FAIL_COND_V_MSG(!is_rotation(), Quaternion(), "Basis must be normalized in order to be casted to a Quaternion. You may be able to normalize a valid rotation Basis by using get_rotation_quaternion() or orthonormalize().");
```

Comment:
Note that `orthonormalize()` isn't binded to GDScript, [`orthonormalized()` is](https://github.com/godotengine/godot/blob/4d08a737fb868b4c2f47946aea7c9ec4ee97894d/core/variant/variant_call.cpp#L1719).

Reply:
`get_quaternion` isn't bound either.